### PR TITLE
Send revenue-relevant events to adjust

### LINKED ISSUE #3445: Send revenue-relevant events to adjust to optimize revenue
**User story**
As a PMM, I would like to have revenue-relevant events available on adjust to analyze/optimize revenue performance of each acquisition channel.

**Acceptance Criteria (how do I know when I’m done?)** 
1. Identify revenue-relevant events
2. Send those events to adjust
3. Verify on adjust in sandbox mode

### PR DESCRIPTION
**Pull request by benjamin-cheng**
Close #3445

### CODE REVIEW

File: app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.kt [hunk 1]
```diff
@@ -63,6 +63,8 @@ object TelemetryWrapper {
     private const val SEARCHCLEAR_TELEMETRY_VERSION = "2"
     private const val SEARCHDISMISS_TELEMETRY_VERSION = "2"
 
+    private val AFFILIATE_SITES = listOf("Bukalapak", "Tokopedia")
```
Comment (cnevinc) [inline app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.kt]: I think "Bukalapak", "Tokopedia" should not be hard-coded.

File: app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.kt [hunk 2]
```diff
@@ -1001,6 +1005,10 @@ object TelemetryWrapper {
 
         EventBuilder(Category.ACTION, Method.ADD, Object.TAB, Value.TOPSITE)
                 .queue()
+
+        if (AFFILIATE_SITES.contains(source)) {
```
Comment (cnevinc) [inline app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.kt]: Please add a comment like "source could be other values" or "we only care about this two source"..

Or maybe just 
```
if (!!TextUtils.isEmpty(source)) {
     AdjustHelper.trackEvent(EVENT_CLICK_AFFILIATE_LINK)
}

```

File: app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.kt (hunk 1 above)
Comment (benjamin-cheng) [reply app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.kt]: Fixed

File: app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.kt (hunk 2 above)
Comment (benjamin-cheng) [reply app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.kt]: Fixed

**Pull request merged**